distributor-node/src/services/parsers/BagIdParserService.ts

```diff
@@ -1,9 +1,24 @@
-import { BagId, DynamicBagTypeKey } from '@joystream/types/storage'
 import { createType } from '@joystream/types'
-import { WorkingGroup, WorkingGroupKey } from '@joystream/types/common'
+import {
+  PalletStorageBagIdType as BagId,
+  PalletCommonWorkingGroup as WorkingGroup,
+  PalletStorageDynamicBagType as DynamicBagType,
+} from '@polkadot/types/lookup'
 import { CLIError } from '@oclif/errors'
 import ExitCodes from '../../command-base/ExitCodes'
 
+export const workingGroups = [
+  'Forum',
+  'Storage',
+  'Content',
+  'OperationsAlpha',
+  'Gateway',
+  'Distribution',
+  'OperationsBeta',
+  'OperationsGamma',
+  'Membership',
+] as const
+
```

Comment:
You can use `keysOf` helper from `@joystream/types`
```suggestion
export const workingGroups = keysOf<PalletCommonWorkingGroup, 'PalletCommonWorkingGroup'>('PalletCommonWorkingGroup')
```

Reply:
addressed in [b17e53e](https://github.com/Joystream/joystream/pull/3997/commits/b17e53e6e1b8c86fb1b87e48195e17e6c3aa38cd)